src/containers/UpdateBookingForm/UpdateBookingForm.container.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import UpdatingBookingFormComponent from "../../components/DentistUpdateBookingForm/DentistUpdateBookingForm.component";
+import { updateBookingDuringTreatment } from "../../services/teeth-apis/BookingController";
+import RequestUpdateFormToRequestUpdateData from "../../mapper/RequestUpdateFormToRequestUpdateData.js";
+import ServicePickingModalContainer from "../ServicePickingModal/ServicePickingModal.container";
+import { notification } from "antd";
+
+const UpdateBookingFormContainer = ({
+  isUpdated,
+  bookingId,
+  setIsRendered,
+  setShowInfo,
```

Comment:
Why do you need setShowInfo

Reply:
To display the alert when waiting for confirm update. This problem is from the previous TEET and you said we could use useRef to handle it but this solution's just fine